pkg/features/features.go

```diff
@@ -58,6 +58,17 @@ const (
 	// owner: @luksa
 	// alpha: v0.1.12
 	ResponseSchema utilfeature.Feature = "ResponseSchema"
+
+	// OriginatingIdentityLocking controls whether we lock OSB API resources
+	// that are being updated while we are still processing the update request.
```

Comment:
`processing the update request` -> `haven't finished processing the current spec`
(and rewrite the whole comment to reflect that)